assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -134,9 +143,36 @@ const List = styled(Area).attrs({
   }
 `;
 
-const Li = styled.li.attrs({
-  role: 'option',
-})`
+const PageSeparator = styled(ReorderableSeparator)`
```

Comment:
This separator is not always perfectly aligned between the pages.

It is kind of clear in this demo, where the separator between pages 1-2 and 2-3 are a tad too far right, whereas the following separators are perfectly aligned in the center between the pages.

![separator-offset](https://user-images.githubusercontent.com/637548/78833628-30099100-79bb-11ea-8f8c-98fa47f4c516.gif)

Reply:
Fixed